.github/workflows/linux.yml

```diff
@@ -48,3 +50,11 @@ jobs:
 
       - name: umockdev test
         run: .private/ci-container-build.sh docker.io/amd64/ubuntu:rolling
+
+      - name: clang-tidy
+        continue-on-error: true
```

Comment:
@tormodvolden I added `continue-on-error: true` here so that CI wont actually fail. the result looks like this
<img width="687" height="233" alt="image" src="https://github.com/user-attachments/assets/0b1767f6-46bc-47f8-bdab-136428f4a026" />

Reply:
Ah OK, that's nice.